fuzzing/repositories.bzl

```diff
@@ -42,3 +42,13 @@ def rules_fuzzing_dependencies():
         ],
         sha256 = "1c531376ac7e5a180e0237938a2536de0c54d93f5c278634818e0efc952dd56c",
     )
+
+    # TODO(sbucur): Make this optional.
```

Comment:
Can you expand comment a little why.

Reply:
Good point, I should have elaborated here properly.